src/app/learn/get-started/advanced/route-params/index.html

```diff
@@ -199,6 +199,14 @@ <h1>About our Team</h1>
 </div>
       </stache-code-block>
     </li>
+    <li>
+      <p>
+        Update index.html to use the about user component selector
```

Comment:
In the current step 8, we say "add index.html to create a page and insert the my-about-user component." and have the same code snippet in place within a longer code sample. It looks like I need to capitalize "I" in "userId" in that code sample, but I think the rest of this new step just repeats step 8.